### PR TITLE
*: support travis fail log setup

### PR DESCRIPTION
**Pull request by siddontang**
@ngaut @BusyJay @disksing

### CODE REVIEW

**Comment by ngaut** [comment]
Cool

**Comment by ngaut** [comment]
LGTM

File: tests/util.rs [hunk 1]
```diff
@@ -128,9 +128,7 @@ impl Drop for CaseTraceLogger {
 // A help function to initial logger.
 pub fn init_log() {
     let output = env::var("LOG_FILE").ok();
-    if output.is_none() {
-        return;
-    }
+
     // don't treat stderr as a filename.
     let output = output.iter().filter(|f| f.to_lowercase() != "stderr").next();
```
Comment (BusyJay) [inline tests/util.rs]: This line can be omitted too.

**Comment by BusyJay** [comment]
I don't think we can add `ci skip` when changing codes.

**Comment by siddontang** [comment]
yep, I remove and re-push again. PTAL

**Comment by BusyJay** [comment]
LGTM

**Pull request merged**